core/src/main/java/io/seata/core/store/db/sql/log/OracleLogStoreSqls.java

```diff
@@ -103,6 +115,13 @@ public String getQueryGlobalTransactionSQLByStatus(String globalTable, String pa
             .replace(PRAMETER_PLACEHOLD, paramsPlaceHolder);
     }
 
+    @Override
+    public String getQueryGlobalTransactionSQLByXidsAndStatus(String globalTable, String xidsParamsPlaceHolder,
+        String paramsPlaceHolder) {
+        return QUERY_GLOBAL_TRANSACTION_BY_XIDS_AND_STATUS_ORACLE.replace(xidsParamsPlaceHolder, XIDS_PLACEHOLD)
```

Comment:
return QUERY_GLOBAL_TRANSACTION_BY_XIDS_AND_STATUS_ORACLE.replace(XIDS_PLACEHOLD,xidsParamsPlaceHolder).replace(PRAMETER_PLACEHOLD,paramsPlaceHolder);

Reply:
add GLOBAL_TABLE_PLACEHOLD

Reply:
code is refactored